README.md

```diff
@@ -2,50 +2,48 @@
 
 UI widgets we use.
 
-## Versioning policy
+## Getting Started
+1. Setup your [development environment](#developing-with-nix)
+2. Run some [tests](#tests)
+3. Check out [some](https://noredink-ui.netlify.app/) [examples](#examples)
```

Comment:
❓ Should this line be `Check out [some examples](https://noredink-ui.netlify.app/)` ?

Reply:
Yeah I think that's better, updated 👍

Reply:
Thanks!